Assets/GothicVR/Scripts/Manager/GvrSceneManager.cs

```diff
@@ -190,5 +206,13 @@ public void MoveToWorldScene(GameObject go)
             GameData.I.WorldScene!.Value.GetRootGameObjects().Append(go);
             SceneManager.MoveGameObjectToScene(go, SceneManager.GetSceneByName(GameData.I.WorldScene.Value.name));
         }
+
+        public void TeleportPlayerToSpot()
+        {
+            var playerParent = generalScene.GetRootGameObjects().FirstOrDefault(go => go.name == "PlayerController");
+
+            if (startPoint != null)
+                playerParent!.transform.Find("VRPlayer").transform.position = startPoint.transform.position;
```

Comment:
Please make "VR player" a constant. If we will rename it or reuse it anywhere else. 😉